### PR TITLE
[#13] reuse open-api-workflows workflows

### PR DESCRIPTION
**Pull request by SonnyBA**
Fixes https://github.com/maykinmedia/open-api-framework/issues/13

Changes

Reuses open-api-workflows workflows to make CI jobs consistent across the different projects.

### CODE REVIEW

**Comment by stevenbal** [comment]
@SonnyBA I noticed that Open Klant does run `lint-oas` (https://github.com/maykinmedia/open-klant/actions/runs/11178494080/job/31076177429?pr=238), but here I don't see it (though it does seem to be configured)

**Review by stevenbal — COMMENTED** [review]
@SonnyBA I noticed that Open Klant does run `lint-oas` (https://github.com/maykinmedia/open-klant/actions/runs/11178494080/job/31076177429?pr=238), but here I don't see it (though it does seem to be configured)

File: src/referentielijsten/api/openapi.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ info:
     name: EUPL
     url: https://github.com/maykinmedia/referentielijsten/blob/master/LICENSE.md
 paths:
-  /api/v1/item:
```
[the remark this answers is not in the document]
Comment (SonnyBA) [reply src/referentielijsten/api/openapi.yaml:13]: I think the spec was changed manually (by hand), see a6b64b77257c90a5842b40ca9ea1f24b57cf3f7f

**Review by annashamray — APPROVED** [review]

File: src/referentielijsten/api/openapi.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (annashamray) [reply src/referentielijsten/api/openapi.yaml:13]: Strange but I don't think we will uncover the truth :)

**Pull request merged**